### PR TITLE
Documented page redirect process

### LINKED ISSUE #276: Document page redirect/renaming process
If a document file (`*.rst`) is renamed, it affects the URI, since doc URIs are just the domain + the file name (without the extension).

A brief section needs to be added to the contrib guide to say that if you rename a file, the PR that renames the file must also add the redirect.

See #273 for how to add a redirect.

### PR DESCRIPTION
**Pull request by ankita240796**
closes #276 

## What is included in this PR?
Added information about redirect process in contributor guide in case of renaming a file.

### CODE REVIEW

**Comment by ankita240796** [comment]
I am sorry for opening and closing multiple PRs. There was some problem with circleci linked with my github account. I revoked it's access and now everything looks fine!

File: contributing.rst [hunk 1]
```diff
@@ -528,6 +528,20 @@ You must include a PR comment. Things to include:
 
 Complete the pull request. The maintainers will review it as quickly as possible. If there are any problems the maintainers can't deal with, they will reach out to you.
 
+.. note::
+
+   If you happen to rename any document file(:file:`*.rst`), then be sure that you add the redirect in your PR.
+
+   To add the redirect go to :file:`s3_website.yml` and uncomment the **redirects:** line. Add all the mapppings from old file name to new file name below the **redirects:** line one by one.
```
Comment (adammichaelwood) [inline contributing.rst]: uncomment all of them one by one?

Comment (ankita240796) [reply contributing.rst]: I meant to say uncomment just the first redirects line and then add the redirects.
selection_002
selection_003

**Comment by ankita240796** [comment]
@adammichaelwood I am unable to understand why the checks have failed again. Can you please guide me here. Thanks!

**Comment by adammichaelwood** [comment]
@ankita240796 I don't think it is your fault.

@yanokwa There's a problem with CircleCI and this PR, but it doesn't seem to be related to the committed file changes, or to anything Sphinx related.

**Comment by yanokwa** [comment]
@ankita240796 Are you still having problems with this when you do a rebuild in CircleCI at https://circleci.com/gh/ankita240796/docs/tree/redirect?

**Comment by ankita240796** [comment]
@yanokwa I have removed my account from circleci already and there was no problem with the new PRs. Shall I close this one, make a new branch and issue a PR again?

**Comment by lognaturel** [comment]
I think we can just merge when it's ready and see if there are any issues on `master`. I would guess not.

File: contributing.rst (hunk 1 above)
Comment (ankita240796) [reply contributing.rst]: @adammichaelwood can you please check this and guide me what changes do I need to make.

Comment (adammichaelwood) [reply contributing.rst]: I would add a sentence, and change a couple words.

> Add a mapping from the old file name to the new file name below the **redirects:** line, one mapping per line. Several examples of how to format these are shown in the comments.

**Comment by ankita240796** [comment]
@adammichaelwood I made the requested changes.

**Pull request merged**